packages/circuit-ui/components/Table/components/TableBody/TableBody.spec.tsx

```diff
@@ -36,7 +38,7 @@ describe('TableBody', () => {
     describe('rowHeaders', () => {
       it('should render a table cell as the first element on each row with no rowHeaders', () => {
         const { getByTestId } = render(<TableBody rows={fixtureRows} />);
-        const tableCell = getByTestId('table-row').firstChild;
+        const tableCell = getByTestId('table-row').children[0];
```

Comment:
TypeScript was unhappy with `firstChild` (not the same type as `children[0]`, otherwise JavaScript wouldn't be fun!)